module/VuFind/src/VuFind/Search/Base/HierarchicalFacetHelperInterface.php

```diff
@@ -44,24 +47,24 @@ interface HierarchicalFacetHelperInterface
      * Helper method for building hierarchical facets:
      * Sort a facet list according to the given sort order
      *
-     * @param array          $facetList Facet list returned from Solr
-     * @param boolean|string $order     Sort order:
+     * @param array            $facetList Facet list returned from Solr
+     * @param bool|string|null $order     Sort order:
```

Comment:
I wonder if we should adjust the comment to note that null will use the default sort order (since that appears to be the implemented behavior).